app/src/main/java/org/eyeseetea/malariacare/presentation/presenters/MediaPresenter.java

```diff
@@ -10,13 +10,13 @@
 
     View mView;
     private GetMediaUseCase mGetMediaUseCase;
-    private IStylePreferencesRepository.ListStyle listStyle;
+    private Settings settings;
     List<Media> mMediaList;
 
     public MediaPresenter(
```

Comment:
The presenter should have the responsibility to execute use case for retrieve data so It's more correct pass SettingsRepository as argument